### ISSUE TITLE
Clean the integration list (Remove Stripe, Airtable, PostgreSQL)

### ISSUE THREAD

**Issue by Bonapara** [twentyhq/twenty#7110]
# Current behavior

In `Settings > Integrations` We display 3 integrations as _coming soon_

![CleanShot 2024-09-18 at 10 02 38](https://github.com/user-attachments/assets/0d50b029-45f5-49d6-b4df-8c6771ddabea)

# Desired Behavior

Remove these three integrations (Airtable, Stripe, PostgreSQL) as they are no longer planned for the short term.

**Comment by RaulErnesto08** [comment]
Hello @Bonapara , I'll like to work on this, can you assign it to me? 😄

**Comment by ehconitin** [comment]
He will assign you soon till then go ahead and raise the PR @RaulErnesto08!
Thanks for contributing

**Comment by Bonapara** [comment]
Sure @RaulErnesto08, thanks for contributing!

**Comment by RaulErnesto08** [comment]
Hey there! Sorry for the late response, I was diving into the code and have a couple of questions.

I noticed there are a few flags: `is[Integration]Enabled` and `is[Integration]Active`. From what I understand, these flags control whether an integration shows as "Soon," "Active," or "Add."

The goal is to remove the Airtable, Stripe, and PostgreSQL integrations, but I'm wondering:

Should I remove these integrations only when they are not enabled (when they are shown as "Soon")?
Or, should I remove them entirely, regardless of whether they are enabled or active?

I was thinking about adjusting the array in `useSettingsIntegrationCategories` by removing `getSettingsIntegrationAll` entirely, like this:

Current:
```typescript
return [
      getSettingsIntegrationAll({
        isAirtableIntegrationEnabled,
        isAirtableIntegrationActive,
        isPostgresqlIntegrationEnabled,
        isPostgresqlIntegrationActive,
        isStripeIntegrationEnabled,
        isStripeIntegrationActive,
      }),
      SETTINGS_INTEGRATION_ZAPIER_CATEGORY,
      SETTINGS_INTEGRATION_WINDMILL_CATEGORY,
      SETTINGS_INTEGRATION_REQUEST_CATEGORY,
    ];
```

New:
```typescript
return [
      SETTINGS_INTEGRATION_ZAPIER_CATEGORY,
      SETTINGS_INTEGRATION_WINDMILL_CATEGORY,
      SETTINGS_INTEGRATION_REQUEST_CATEGORY,
    ];
```

Does this approach make sense? or is there a better way to handle this?

I'm sorry if this questions seems too obvious, is my first time contributing to this project and I want to make sure I'm following the best practices.

Thanks in advance!

**Comment by ehconitin** [comment]
@Bonapara,
Just to clarify, are we looking to completely remove these integrations, or should we simply disable them so that they no longer render?

@RaulErnesto08, is this aligned with what you were suggesting?

**Comment by Bonapara** [comment]
Thank you both for your messages. For V1, we can limit the removal to when they are displayed as "Soon" only, allowing users with an activated flag to access the integration settings.

Does it make sense to you @ehconitin @RaulErnesto08?

**Comment by ehconitin** [comment]
Yep makes sense!

> Should I remove these integrations only when they are not enabled (when they are shown as "Soon")?

@RaulErnesto08 Yes!

**Comment by ehconitin** [comment]
closed in #7259
